salt/digests/config/home-deployuser-digests-containers.env

```diff
@@ -0,0 +1,9 @@
+ENVIRONMENT_NAME={{ pillar.elife.env }}
+NEW_RELIC_MONITOR_MODE={{ 'true' if pillar.elife.newrelic.enabled else 'false' }}
+NEW_RELIC_APP_NAME={{ salt['elife.cfg']('project.stackname') }}
+NEW_RELIC_LICENSE_KEY={{ pillar.elife.newrelic.license }}
+NEW_RELIC_LABELS=project:{{ salt['elife.cfg']('project.project_name', 'python-unknown-project') }};env:{{ pillar.elife.env }}
+AWS_DEFAULT_REGION={{ pillar.elife.aws.region }}
+AWS_ACCESS_KEY_ID={{ pillar.elife.aws.access_key_id }}
+AWS_SECRET_ACCESS_KEY={{ pillar.elife.aws.secret_access_key }}
+PYTHONOPTIMIZE=1
```

Comment:
if this is related to `python -O ...`, then I wonder if we should run the optimised code through the tests. Obviously `assert`s will be stripped out.

https://docs.python.org/2.7/tutorial/modules.html#compiled-python-files

Reply:
Will verify for this project, but `pytest` should take care of avoiding that https://docs.pytest.org/en/reorganize-docs/faq.html#what-s-this-magic-with-pytest-historic-notes

Reply:
ha - I didn't know pytest had been around so long. Django also had a 'magic removal' branch about the same time as pytest's own. Rewriting `assert` statements makes sense for a test runner.